debian/source/format

```diff
@@ -0,0 +1 @@
+3.0 (quilt)
```

Comment:
well not quite, if you merge into master, the packaging becomes bundled with code, and you should use format "3.0 (native)", this would require you to change the changelog version to "1.0" as well (no "-")

Reply:
You are right, thanks to notice this little mistake i made.